packages/ui-react/src/components/Payment/Payment.tsx

```diff
@@ -159,58 +115,39 @@ const Payment = ({
     }
   }
 
-  const showChangeSubscriptionButton = (!isExternalPaymentProvider && offerSwitchesAvailable) || (!isChangingOffer && !canRenewSubscription);
+  const showChangeSubscriptionButton = !isExternalPaymentProvider && offerSwitchesAvailable;
```

Comment:
Maybe we should also add `canChangeSubscription` feature flag to hide the button even though we get some offers?